src/components/search-bar.tsx

```diff
@@ -0,0 +1,43 @@
+import { useFormContext } from "react-hook-form";
+import { FormControl, FormField, FormItem } from "./ui/form";
+import { Input } from "./ui/input";
+
+export function SearchBar() {
+  const form = useFormContext();
+
+  return (
+    <FormField
+      control={form.control}
+      name="searchText"
+      render={({ field }) => (
+        <FormItem>
+          <FormControl>
+            <div className="relative w-80">
+              {/* SEARCH TEXT INPUT */}
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                className="absolute bottom-0 left-3 top-0 my-auto h-6 w-6 text-gray-500"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke="currentColor"
+              >
+                <path
+                  strokeLinecap="round"
+                  strokeLinejoin="round"
+                  strokeWidth={2}
+                  d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
+                />
+              </svg>
+
+              <Input
```

Comment:
Looks great! Just wondering if there is a need for a submit button since the only way to submit right now is by pressing the Enter key right?

Reply:
I left it out b/c it wasn't in the design but maybe we can make a small pr in the future to add it